Change the way ready containers find worker pods (#180)
grpc/test-infra: containers/init/ready/README.md

# Ready

> Ready is a container that waits for a list of pods within a loadtest to

Author: Done

become available. It exits successfully when all worker pods are ready, writing a
comma-separated list of their IP addresses to a file. It exits unsuccessfully if
a timeout was exceeded before all pods were ready.

## Usage

The container relies on command line argument to specify the load test's name.

> For example,
> 
> ```shell
> go run ready.go LOADTEST_NAME
> ```
> 
> Will wait for all worker pods which belongs to LOADTEST_NAME

Reviewer: nit: since there is no longer a list, it may read better if we merge lines 11 and 17:

````suggestion
For example, the following waits for all worker pods which belong to the LoadTest named `<LOADTEST_NAME>`:

```shell
go run ready.go <LOADTEST_NAME>
```
````

Also, this doesn't have to be fixed in this PR (and I apparently broke the convention myself (sorry))... but generally command line arguments adopt the following conventions:

- Angle brackets, such as `<ARG-NAME>`, indicate the argument is required.
- Square brackets, such as `[ARG-NAME]`, indicate the argument is optional.

You will sometimes see no angle brackets, such as the [NetBSD style](https://en.wikipedia.org/wiki/Usage_message#Examples) or the [Google Developer documentation](https://developers.google.com/style/code-syntax). However, the Google developer documentation switches the color and font for required variables. gcloud uses the angle brackets on the command line. If we don't, it can be hard to determine what is a command and what is a required argument. Most of the tools we use rely on angle brackets. For example,

- `git -h`:
  ```
  usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           [--super-prefix=<path>] [--config-env=<name>=<envvar>]
           <command> [<args>]
  ```

- `gcloud compute project-info -h`:
  ```
   Usage: gcloud compute project-info [optional flags] <command>
       command may be             add-metadata | describe | remove-metadata |
                                  set-usage-bucket | update
  ```

The Python [docopt](http://docopt.org/) project provides some good conventions. They actually generate a command line argument and flag parser using your usage string documentation.

Author: Thanks for providing extra information! Very interesting :)

Reviewer 2: FYI, conventions that are good for usage strings may not be good for markdown documentation. It is not important here because we can always fix it later, so let's merge the change and work on a release.
